RecoTracker/MkFitCMS/src/MkStdSeqs.cc

```diff
@@ -0,0 +1,586 @@
+#include "RecoTracker/MkFitCMS/interface/MkStdSeqs.h"
+
+#include "RecoTracker/MkFitCore/interface/HitStructures.h"
+#include "RecoTracker/MkFitCore/interface/IterationConfig.h"
+
+#include "RecoTracker/MkFitCore/interface/binnor.h"
+
+#include "tbb/parallel_for.h"
+
+namespace mkfit {
+
+  namespace StdSeq {
+
+    //=========================================================================
+    // Hit processing
+    //=========================================================================
+
+    void LoadDeads(EventOfHits &eoh, const std::vector<DeadVec> &deadvectors) {
+      for (size_t il = 0; il < deadvectors.size(); il++) {
+        eoh.SuckInDeads(int(il), deadvectors[il]);
+      }
+    }
+
+    // Loading hits in CMSSW from two "large multi-layer vectors".
+    // orig_hitvectors[0] - pixels,
+    // orig_hitvectors[1] - strips.
+
+    void Cmssw_LoadHits_Begin(EventOfHits &eoh, const std::vector<const HitVec *> &orig_hitvectors) {
+      eoh.Reset();
+
+      for (auto &&l : eoh.m_layers_of_hits) {
+        // This slightly sux.
```

Comment:
please improve this comment

Reply:
Addressed in Round 1 follow-up PR.
trackreco/cmssw#75 https://github.com/trackreco/cmssw/pull/75/commits/e9834b4a33dab0a34078a4861b825f0e1ebdb64d